src/go/k8s/tests/e2e/produce-consume/00-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: apps/v1
```

Comment:
we should think about how to do this in a test setup somehow because otherwise we'll be duplicating this in every test. I am fine with this being a follow-up

Reply:
You are right. Will do that as follow-up